### DISCUSSION TITLE
Python repository expertise analyser - Academical Study

### DISCUSSION THREAD

**Discussion by nikogio** [community/community#39849] [GitHub Education]
Expertise can be a very fluffy term, especially when developing softwares.

For research purposes as part of my Advanced Software Engineering, we have created a survey to analyse usage expertise in Python projects.

If you happen to have any pubblic Python repositories on Github, and feel like contributing with 5 minutes of your time, you can participate and read more here👇🏼

https://victorious-cliff-0fddac003.2.azurestaticapps.net/

**Comment by samus-aran** [comment]
Hi @Nikogio, we’re glad you’re here 😀

Thanks for sharing your survey with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
